package.json

```diff
@@ -93,7 +93,7 @@
   "author": "Artem Zakharchenko",
   "license": "MIT",
   "engines": {
-    "node": ">=18"
+    "node": ">=18.20.0 <20 || >=20.12.1"
```

Comment:
Why skipping v21? 

Also, could you give more context why we need to update the version?

Reply:
Because of the `rawHeaders` symbol change in Node.js.

> Why skipping v21?

What do you mean?
This range translates to: "any version between 18.20.0 and 20, OR anything above 20.12.1"
You can play this it [here](https://jubianchi.github.io/semver-check/#/constraint/%3E%3D18.20.0%20%3C20%20||%20%3E%3D20.12.1)

Reply:
Still a bit confused. We currently don't depend on Node.js for raw headers and use our own proxy to gather them. Does the version change still apply in that case?

Reply:
Interesting..
```js
const { ClientRequestInterceptor } = require('@mswjs/interceptors/ClientRequest')
const http = require('http')

const interceptor = new ClientRequestInterceptor({
  name: 'my-interceptor',
})
interceptor.apply();
interceptor.on('request', async ({ controller }) => {
  controller.respondWith(new Response(null, { headers: { 'set-cookie':  ['mocked-cookie-0;mocked-cookie-1', 'mocked-cookie-2'] }}))
});

(async function () {
   http.get('http://example.com', (res) => {
    console.log(res.headers);
   });
})()
```

With version 20.11.1 the output is:
> { 'set-cookie': [ 'mocked-cookie-0;mocked-cookie-1,mocked-cookie-2' ] }

With version 20.12.2 the output is:
> {  'set-cookie': [ 'mocked-cookie-0;mocked-cookie-1', 'mocked-cookie-2' ] }